src/libslic3r/GCode/PressureEqualizer.cpp

```diff
@@ -619,8 +558,7 @@ void PressureEqualizer::adjust_volumetric_rate(const size_t fist_line_idx, const
                 // Limit by the succeeding volumetric flow rate.
                 rate_end = rate_succ;
 
-            // don't alter the flow rate for these extrusion types
-            if (!line.adjustable_flow || line.extrusion_role == GCodeExtrusionRole::BridgeInfill || line.extrusion_role == GCodeExtrusionRole::Ironing) {
+            if (!line.adjustable_flow || line.extrusion_role == GCodeExtrusionRole::ExternalPerimeter || line.extrusion_role == GCodeExtrusionRole::GapFill || line.extrusion_role == GCodeExtrusionRole::BridgeInfill || line.extrusion_role == GCodeExtrusionRole::Ironing) {
```

Comment:
@igiannakas 
One of the difference is that PS won't apply Pressure Eq to ExternalPerimeter and GapFill either.
GapFill is understandable.

I'm not too sure about why don't apply to ExternalPerimeter thought.
You have done the extensive tests on your side, I'll leave it to you to make the judgement call whether to execlude external perimeters.

Reply:
I’d think it’s best to leave the external perimeter in the code, it’s where the most benefit is :) - let me test without the changes from the above user but with external perimeter enabled and I’ll revert back.